src/Javis.jl

```diff
@@ -191,8 +192,12 @@ function render(
     frames = preprocess_frames!(objects)
 
     if liveview == true
-        _javis_viewer(video, length(frames), objects)
-        return "Live preview started."
+        if isdefined(Main, :IJulia) && Main.IJulia.inited
+            return not_repl(video, length(frames), objects)
```

Comment:
I would maybe rename the function to `notebook_viewer`

Reply:
I agree. And since we want to also support Pluto notebooks, perhaps a name like `_jupyter_viewer`.